src/modules/FSXARemoteApi.ts

```diff
@@ -209,6 +210,14 @@ export class FSXARemoteApi implements FSXAApi {
     if (filters) {
       let localeFilter: QueryBuilderQuery[] = []
       if (locale) {
+        if (typeof locale !== 'string' || !locale.includes('_')) {
+          this._logger.warn(
+            '[buildNavigationServiceUrl]',
+            'Parameter "locale" has been given.'
```

Comment:
I am not sure what this log message is supposed to express but I believe it does not express what the code does.

The message suggests that the function that was called has successfully received a parameter "locale" and then it throws an error.
If the message should in fact confirm, that a parameter "locale" was passed to the function as is checked in line 212 then the message should be logged there? If the message is supposed to express something different as is suggested by logger.warn, then it needs to be reworded.